tests/test_redir.py

```diff
@@ -303,7 +303,7 @@ def test_very_long_output(test_settings, capfd, runner, console, numpy):
             print("LOG" * 1000000)
             print("\x1b[31m\x1b[40m\x1b[1mHello\x01\x1b[22m\x1b[39m" * 100)
             print("===finish===")
-            time.sleep(3)
+            time.sleep(5)
```

Comment:
this remove a local flake I was seeing.

Reply:
If 3 wasn't good, 5 will definitely always work 🤣 .  Fear not this isn't the only place we do this, I forgive you.